src/main/flight/position.c

```diff
@@ -219,28 +211,31 @@ void calculateEstimatedAltitude(void)
     DEBUG_SET(DEBUG_ALTITUDE, 3, estimatedVario);
 #endif
     DEBUG_SET(DEBUG_RTH, 1, lrintf(displayAltitudeCm / 10.0f));
+    DEBUG_SET(DEBUG_ALTHOLD, 1, lrintf(zeroedAltitudeCm));
+    DEBUG_SET(DEBUG_GPS_RESCUE_THROTTLE_PID, 1, lrintf(zeroedAltitudeCm));
 
     altitudeAvailable = haveGpsAlt || haveBaroAlt;
 }
+
 #endif //defined(USE_BARO) || defined(USE_GPS)
 
-bool isAltitudeAvailable(void) {
-    return altitudeAvailable;
+float getAltitudeCm(void)
```

Comment:
It would be great to use return value in meters here. Altitude related code would be fixed with lot of *100 and /100, which are easily visible and may be eventually factored out (and Cm suffixes removed).
This is legacy from old integer-only days, it is only encumbering code now.

Also, it is a easier to visualize meters and m/s.

Reply:
True, though since all these tasks run at 100Hz, and we measure in cm, a 1m/s velocity is a 1.0 cm altitude change per iteration, and that's also easy enough to visualise, for me, anyway.

However, you're right, this should be done at some point.